facility-manager.html

```diff
@@ -119,6 +122,15 @@
 </p>
 
 <p>
+    Administrators with restricted privileges can have varying levels
+    of specific permissions, see <a href="https://www.openmicroscopy.org/latest/omero/sysadmins/admins-with-restricted-privileges.html"
+    target="_blank" class="external-url">Administrators with restricted privileges</a>.
```

Comment:
I'd add 'sysadmin documentation' here to make clear that they are leaving the help docs and going into the technical docs when they click